### DISCUSSION TITLE
Null Values, and Numeric Field Limits

### DISCUSSION THREAD

**Discussion by cilerler** [etsy/open-api#965] [Ideas]
As a developer who is currently consuming the Etsy API through C#, I have encountered a few challenges related to the documentation that I would like to bring to your attention:

1. **Null Values**: It would be extremely helpful if the API documentation could explicitly indicate whether a particular field can return a null value or not. As of now, it is unclear from the documentation which fields may contain null values, and this ambiguity is making it challenging to create the necessary objects in C# without encountering null reference exceptions.

2. **Number Field Limits**: For fields that return numeric values, the documentation does not specify the maximum value that can be returned. The lack of this information makes it difficult to determine the appropriate data type to use in C# when parsing these values. I believe instead of using 0 or 1 in examples, specifying the maximum value for each numeric field would greatly improve the experience of consuming the API.

**Comment by Gareth064** [comment]
It's been brought up a few times that the ints need to be changed to bigint or long etc... But at the time Etsy didn't think it was possible. But we have provided evidence that the OpenAPI spec supports it. Still nothing done about it though.
